doc/release/upcoming_changes/26724.new_feature.rst

```diff
@@ -0,0 +1,6 @@
+* `numpy.cumulative_sum` and `numpy.cumulative_prod` were added as Array API
+  compatible alternatives for `numpy.cumsum` and `numpy.cumprod`.
```

Comment:
Is it useful to mention the differences? I.e., that one cannot pass in an `initial`, but can `include_initial` in the result?

Reply:
I added one line about `include_initial` functionality. Done!